test/compilable/extra-files/rdepsOutput.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+grep 'rdeps7016 (.*rdeps7016.d) : private : rdeps7016a' ${RESULTS_DIR}/compilable/rdeps7016.deps || exit 1
```

Comment:
Does this actually require a shell test?  If possible, it should make use of the TEST_OUTPUT sections which automatically match the compiler output.